files/build_templates/pmon.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
@Junchao-Mellanox This will affect the fast-reboot time for link bring up on platform where Xcvrd is responsible for link bring up.